### ISSUE TITLE
Offsets committed on every Poll call

### ISSUE THREAD

**Issue by kapoorrraghav** [confluentinc/confluent-kafka-dotnet#361]
Description
===========
I have disabled auto commit in Consumer. I want to commit offsets manually in Kafka and expected that unless I explicitly call consumer.CommitAsync(message) the offsets will never be committed. But, I am observing that offsets are committed on every consumer Poll() call. This doesn't sound correct, is this the normal behaviour? If yes, then what's the significance of committing offsets manually periodically?

Also, I can see the above behaviour with Auto Commit on as well. 

How to reproduce
================
Able to see this behaviour even in AdvancedConsumer Console app shipped with Confluent.Kafka .Net code. 
**Configuration:**         
            new Dictionary<string, object>
            {
                { "group.id", "advanced-csharp-consumer" },
                { "statistics.interval.ms", 60000 },
                { "bootstrap.servers", brokerList },
                { "default.topic.config", new Dictionary<string, object>()
                    {
                        { "auto.offset.reset", "latest" }
                    }
                }

**OnOffsetsCommitted** is raised - attached handler is invoked after the message is processed consumer.Poll() is called. 

Checklist
=========
Please provide the following information:

 - [ ] Confluent.Kafka nuget version: 0.11.2
 - [ ] Apache Kafka version: 
 - [ ] Client configuration: Please refer How to reproduce section
 - [ ] Operating system: Windows 8
 - [ ] Provide logs (with "debug" : "..." as necessary in configuration)
 - [ ] Provide broker log excerpts
 - [ ] Critical issue

**Comment by kapoorrraghav** [comment]
Looks like "enable.auto.commit" is expected to be explicitely set to false.

**Comment by kapoorrraghav** [comment]
Please ignore this query. It works as expected. I removed the configuration setting of "enable.auto.commit" to manually manage offset instead of setting it to false.

**Comment by mhowlett** [comment]
correct, but note that `auto.commit.interval.ms` sets the frequency at which offsets are automatically committed in the case `enable.auto.commit` is true.
